src/providers/authenticationProvider.ts

```diff
@@ -104,17 +107,68 @@ export class TerraformCloudAuthenticationProvider implements vscode.Authenticati
   async createSession(_scopes: readonly string[]): Promise<vscode.AuthenticationSession> {
     this.ensureInitialized();
 
-    // Prompt for the UAT.
-    const token = await vscode.window.showInputBox({
-      ignoreFocusOut: true,
-      placeHolder: 'User access token',
-      prompt: 'Enter an HashiCorp Terraform User Access Token (UAT).',
-      password: true,
-    });
+    vscode.window.showInformationMessage;
+    const choice = await vscode.window.showQuickPick(
+      [
+        {
```

Comment:
Should we hide this choice for non-native (e.g. web) platforms? Or do we skip supporting TFC on the web for the MVP?

Reply:
We're going to have to skip web for now as TFC API does not accept CORS at the moment, last I checked.